src/GroupFactory.php

```diff
@@ -7,25 +7,8 @@
 
 final class GroupFactory
```

Comment:
Is this class used anywhere?

Reply:
Yes, see my the first post of my fastroute PR